src/test/integration/django/conftest.py

```diff
@@ -5,4 +5,4 @@
 
 @pytest.fixture(autouse=True)
 def increment_spans_counter():
-    SpansContainer.update_span_offset()
+    SpansContainer.reset_spans_file()
```

Comment:
IMO the span dump files shouldn't be shared between tests, this makes debugging and span parsing unnecessarily complex and painful. this is a workaround, and a recommendation to refactor one day if there's time...